clearpath_sensors/config/fixposition.yaml

```diff
@@ -0,0 +1,16 @@
+fixposition_driver:
+  ros__parameters:
+    fp_output:
+      formats:
+        - ODOMETRY
+        - LLH
+        - RAWIMU
+        - CORRIMU
+      port: "21001"
+      ip: 192.168.131.35
+      rate: 200
+      type: tcp
+      reconnect: 5
+    customer_input:
+      speed_topic: "xvn/speed"
```

Comment:
Should we try keep these along the same lines as our API?  Something like `/sensors/ins/input/speed`?

Reply:
We could, but according to the outdoor nav team we aren't even using this topic right now. And it may be very specific to this particular driver. I'm okay leaving it as-is for now, but we can revisit this in the future if/when we add more INS devices to see where they overlap in terms of parameters and topics.